packages/@aws-cdk/aws-ec2-alpha/test/subnet-v2.test.ts

```diff
@@ -207,7 +207,7 @@ describe('Subnet V2 with custom IP and routing', () => {
                     },
```

Comment:
You need to add unit test cases that cover `fromSubnetV2Attributes` function

Reply:
added https://github.com/aws/aws-cdk/pull/31765/files#diff-270676bd6ded7f6c3a90d535b610f1e8f5d6700141da407b88b0bad5e1850bf7R201